tests/cycle.rs

```diff
@@ -882,7 +882,6 @@ fn cycle_unchanged() {
         [
             "salsa_event(DidValidateMemoizedValue { database_key: min_iterate(Id(1)) })",
             "salsa_event(DidValidateMemoizedValue { database_key: min_panic(Id(2)) })",
-            "salsa_event(DidValidateMemoizedValue { database_key: min_iterate(Id(1)) })",
```

Comment:
Can you explain why we see behavior changes here?

Reply:
This is removing a redundant verification (`min_iterate(1)` was already verified above), which is the point of the change as I understand it